gap/Forms.gi

```diff
@@ -164,6 +164,61 @@ function(group, type)
     return ConjugateToSesquilinearForm(group, broadType, gapForm);
 end);
 
+# Let <dims> = [d1, d2, ..., dt], consider the tensor product of the vector
+# spaces F ^ d1, F ^ d2, ..., F ^ dt and in it the i-th basis vector ei. 
+#
+# Return a list of indices [i1, i2, ..., it] such that ei is the tensor product 
+# of the basis vectors ei1, ei2, ..., eit of F ^ d1, F ^ d2, ..., F ^ dt.
```

Comment:
However, after writing this, and thus finally understanding what it does, I am not even convinced it should be kept. I think it would be more efficient and easier to maintain if the code below was changed to use `IteratorOfCartesianProduct(List(dims,d->[1..d]))`.

Alas, I don't want to require either of you here; such a change could be made later, if at all.

Reply:
I think the choice is not as arbitrary as it might seem: I believed it to be quite common to mean by `e_i` the `i`-th standard basis vector (i.e. with a `1` in the `i`-th component and zeroes elsewhere). Then each `e_i` can be uniquely written as some *Kronecker product* `e_i1 x e_i2 x ... x e_it` and the function returns exactly the indices `i1, ..., it`. 

So I guess the wording of the comment should be changed slightly so that it states we are assuming `e_i` to be the `i`-th standard basis vector and that we "choose" the tensor product of a collection of vectors to be their Kronecker product?

Reply:
But you are referring with ei to two different things: one is the i-th standard basis vector in K^n, and I agree this is common enough. But then in line 170 you use it to refer to a basis vector in the Kronecker product, and there this is *not* clear. My main point is that the *order* of tuples is arbitrary, and should be documented

Reply:
I left it as-is since I propose to replace this code anyway, see PR #95